specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-03-01/managedClusters.json

```diff
@@ -4398,6 +4441,66 @@
           "description": "The type of the extended location."
         }
       }
+    },
+    "OSOptionProperty": {
+      "properties": {
+        "os-type": {
+          "type": "string",
+          "description": "OS type."
+        },
+        "enable-fips-image": {
+          "type": "boolean",
+          "description": "Whether FIPS image is enabled."
+        }
+      },
+      "required": [
+        "os-type",
+        "enable-fips-image"
+      ],
+      "description": "OS option property."
+    },
+    "OSOptionPropertyList": {
+      "properties": {
+        "osOptionPropertyList": {
```

Comment:
`osOptionPropertyList` shows up in payload and would be in SDKs. Suggest `osOptions` or `options`

Reply:
This name has been used in the server side too, so better keep it to be consistent with server side.

https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=%2Fresourceprovider%2Fserver%2Fmicrosoft.com%2Fcontainerservice%2Fdatamodel%2Fv20210301%2Ftypes.go&version=GBmaster&line=753&lineEnd=753&lineStartColumn=2&lineEndColumn=22&lineStyle=plain&_a=contents